roles/keystone/defaults/main.yml

```diff
@@ -33,6 +33,13 @@ keystone_keycloak_admin_user: admin
 keystone_keycloak_admin_password: "{{ keycloak_admin_password }}"
 keystone_keycloak_realm: atmosphere
 keystone_keycloak_realm_name: Atmosphere
+keystone_keycloak_realm_default_password_policy: "length(15) and upperCase(1) and lowerCase(1) and digits(1) and specialChars(1) and notEmail() and notUsername() and passwordHistory(10) and forceExpiredPasswordChange(180)"
+keystone_keycloak_realm_default_brute_force_protected: true
+keystone_keycloak_realm_default_brute_force_failure_factor: 15
+keystone_keycloak_realm_default_brute_force_wait_increment_seconds: 1800
+keystone_keycloak_realm_default_brute_force_max_failure_wait_seconds: 3600
+keystone_keycloak_realm_default_brute_force_max_delta_time_seconds: 3600
+keystone_keycloak_realm_default_totp_default_action: true
```

Comment:
I think we can disable in default and just users can enable if they want?
@mnaser what is your opinion?